beacon-chain/gateway/server/main.go

```diff
@@ -21,6 +21,7 @@ var (
 	debug                   = flag.Bool("debug", false, "Enable debug logging")
 	allowedOrigins          = flag.String("corsdomain", "", "A comma separated list of CORS domains to allow")
 	enableDebugRPCEndpoints = flag.Bool("enable-debug-rpc-endpoints", false, "Enable debug rpc endpoints such as /eth/v1alpha1/beacon/state")
+	grpcMaxMsgSize          = flag.Int("grpc-max-msg-size", 1<<22, "Integer to define max recieve message call size (default: 4194304 (for 40MB))")
```

Comment:
This is going to print weird since flag --help will print the default value

Reply:
got it, thanks!